src/repository/models.py

```diff
@@ -451,16 +451,25 @@ def display_authors(self):
         return ", ".join([author.full_name for author in self.authors])
 
     def add_user_as_author(self, user):
+
         author_dict = {
             'first_name': user.first_name if user.first_name else '',
             'middle_name': user.middle_name if user.middle_name else '',
             'last_name': user.last_name if user.last_name else '',
             'affiliation': user.affiliation() if user.affiliation else '',
+            'orcid': user.orcid if user.orcid else '',
         }
+
         author, a_created = Author.objects.get_or_create(
             email_address=user.email,
             defaults=author_dict,
         )
+
+        #if the author object already exists, but does not have an orcid, the default given above will not be used... solution: go back and update the orcid
```

Comment:
Strictly speaking this should be `# if the author`

Reply:
I double-checked my VSCode config, it's set up to use autopep8, and autopep8 is installed. Oddly enough, this space after a hash issue is not flagged. But, I trust you, and have made the requested change.